Rivet/Rivet.psd1

```diff
@@ -11,7 +11,7 @@
     RootModule = 'Rivet.psm1'
 
     # Version number of this module.
-    ModuleVersion = '0.13.0'
+    ModuleVersion = '0.14.0'

```

Comment:
Let's also move all the release notes to `CHANGELOG.md` file.
1. Rename the existing `RELEASE_NOTES.txt` file to `CHANGELOG.md`
2. Go through the all the Rivet releases [on the PowerShell Gallery](https://www.powershellgallery.com/packages/Rivet/) and add the release notes for all versions whose release notes are missing from the `CHANGELOG.md` file.
3. Update the ReleaseNotes property in this file to be a link to the `CHANGELOG.md` file on GitHub.